app/grandchallenge/products/utils.py

```diff
@@ -92,7 +92,7 @@ def _create_product(self, row, c):
         p.product_name = row["Product name"]
         p.company = c
         p.modified = row["Timestamp"]
-        p.short_name = row["Short name"][:50]
+        p.slug = row["Short name"][:50]
```

Comment:
might be good to use `slugify` for this as well, just in case a space or something accidentally sneaks in there.